src/main/java/com/softserve/academy/spaced/repetition/controller/CardsFileController.java

```diff
@@ -1,71 +1,46 @@
 package com.softserve.academy.spaced.repetition.controller;
 
 import com.softserve.academy.spaced.repetition.service.CardService;
-import com.softserve.academy.spaced.repetition.service.cardLoaders.impl.CardLoadService;
 import com.softserve.academy.spaced.repetition.utils.exceptions.EmptyFileException;
 import com.softserve.academy.spaced.repetition.utils.exceptions.NotAuthorisedUserException;
 import com.softserve.academy.spaced.repetition.utils.exceptions.NotOwnerOperationException;
 import com.softserve.academy.spaced.repetition.utils.exceptions.WrongFormatException;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.*;
 import org.springframework.web.multipart.MultipartFile;
 
 import javax.servlet.http.HttpServletResponse;
 import java.io.IOException;
-import java.sql.SQLException;
-import java.util.NoSuchElementException;
 
 @Controller
 public class CardsFileController {
 
     @Autowired
     private CardService cardService;
 
-    @Autowired
-    private CardLoadService cardLoadService;
-
-    @PostMapping("api/private/upload/deck/{deckId}/cards")
-    public ResponseEntity uploadFile(@RequestParam("file") MultipartFile cardsFile, @PathVariable Long deckId)
-            throws WrongFormatException,NotOwnerOperationException, NotAuthorisedUserException, EmptyFileException,
-            IOException {
+    @PostMapping("api/upload/deck/{deckId}/cards")
+    @ResponseStatus(HttpStatus.OK)
```

Comment:
Maybe better to return status `CREATED` or `NO_CONTENT`, but not sure)
Or return list of cards, that was uploaded)

Reply:
Must be status 2** , one oh them - OK